src/main/kotlin/no/nav/syfo/melding/kafka/legeerklaring/KafkaLegeerklaringConsumer.kt

```diff
@@ -95,7 +96,11 @@ class KafkaLegeerklaringConsumer(
             meldingType = MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING,
             arbeidstakerPersonIdent = arbeidstakerPersonIdent
         ).lastOrNull()
-        if (utgaaende != null && utgaaende.tidspunkt > OffsetDateTime.now().minusMonths(2)) {
+        val incomingTidspunkt = kafkaLegeerklaring.legeerklaering.signaturDato.toOffsetDateTime()
+        if (
+            utgaaende != null && utgaaende.tidspunkt.toLocalDate() <= incomingTidspunkt.toLocalDate() &&
+            utgaaende.tidspunkt > OffsetDateTime.now().minusMonths(2)
```

Comment:
Er det riktig forstått at vi her tenker at hvis det er 2 måneder eller mer siden en veiledere sendte forespørsel sist, er innkommende legeerklæring uansett ikke relevant?

Finnes det en offisiell "frist" for hvor lang tid en behandler har på seg til å svare? (Det vil jo uansett bare være en nedre grense da, så det har kanskje ikke så mye å si 🤔)

Reply:
Ja, det er riktig oppfattet: Jeg tenkte det kunne være et utgangspunkt. Mener jeg har hørt at det er 14-dagers frist på å svare på en forespørsel om tilleggsopplysninger, om det er det samme for legeerklæring må vi forhøre oss om.